python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -665,11 +647,14 @@ def updateStatus(self, newstatus, transition_update=True, update_master=True, ig
             # move to the new state AFTER hooks are called
             self.status = newstatus
             self._commit()
+            logger.debug("Status changed from '%s' to '%s'" % (saved_status, self.status))
 
         except Exception as x:
             self.status = saved_status
             log_user_exception()
             raise JobStatusError(x)
+        finally:
+            pass
```

Comment:
This was added in debugging and makes life easier when I need to comment out the Exception handling to get a stack trace.
I'm happy to revert but this is effectively used as a debug handle so I'll just comment it out